Doc/tutorial/appendix.rst

```diff
@@ -31,6 +31,26 @@ Typing an interrupt while a command is executing raises the
 :exc:`KeyboardInterrupt` exception, which may be handled by a :keyword:`try`
 statement.
 
+.. _special-var:
+
+Special ``_`` variable
+-------------------------
+
+There is a special variable ``_`` in interactive mode. It stores the result of the 
+last evaluation (*except* ``None``), stored in the :mod:`builtins` module.
```

Comment:
I think "last evaluation" should be "last evaluated expression".

The "except None" part should be clearer, and say that _ isn't changed if the last evaluated expression evaluated to None.

I'm not sure what the part about builtins is trying to say. Can you clarify?

And with this change, we probably need a NEWS entry, so I'm going to remove the skip news label.